src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -166,7 +166,7 @@ export class MeetingManager implements AudioVideoObserver {
     }
   }
 
-  initializeMeetingManager(): void {
```

Comment:
We do want to change the name of this method as this is a public method hence lets not change it for now, but only change the meeting status.

Reply:
oh yeah youre right

Reply:
Hmm I don';t think this is we discussed. I think we propose to create a new (please private) function reset/tearDownMeetingManager which does the samething as `intializeMeetingManager` except publish status.
`initializeMeetingManager` can just call `resetMeetingManager` and publish status.